proposal-template.md

```diff
@@ -1,56 +1,47 @@
 # Project Name
 _Include the name of project being proposed._
 
+# Preferred Maturity Level
+_Specify the stage that you want this project to enter. See [project lifecycle](https://tac.openwallet.foundation/governance/project-lifecycle/#stages) for information on the different stages._
+
 # Project Description
-_Include a description of the project, including what it does, why it is valuable, the origin and history, and other information that will allow others to understand what the project will include._
+_Include a description of the project, including what it does, why it is valuable, the origin and history, and other information that will allow others to understand what the project will include. Architecture diagrams should be included if available._
 
 # Alignment with the OpenWallet Foundation Mission
-_Include a statement on alignment with the [OpenWallet Foundation mission](https://openwallet-foundation.github.io/tac/governance/charter/)._
+_Include a statement on alignment with the [OpenWallet Foundation mission](https://tac.openwallet.foundation/governance/charter/)._
 
-# Current Code of Conduct
-_Has this project adopted a Code of Conduct? If so, please include a link to it._
+# Code of Conduct
+_Has this project adopted a Code of Conduct? If so, please include a link to it.If a current code of conduct is not adopted, will you adopt the [OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)?_
 
 # TAC Sponsor
 _Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
 
 # Project License
-_Include the proposed project license (Apache 2.0 by default)._
+_Include the proposed project license. The OpenWallet Foundation will consider permissive open source licenses and prefers Apache 2.0._
 
 # Source Control
-_If the project already exists, please provide a link to the repositories._
+_If the project already exists, please provide a link to the repositories. If no source exists, this is okay. Please state that this will be a completely new project with code beginning in the OpenWallet Foundation._
 
 # Issue Tracker
-_If the project already exists, please provide a link to the issue tracker._
+_If the project already exists, please provide a link to the issue tracker. If there is no issue tracker, please answer with "N/A (not applicable)"._
```

Comment:
@tkuhrt This help text refers explicitly to issue tracking that already exist. What should a project that does not track issues put here?

Do we implicitly expect  that after the adoption and the move under OWFs GitHub the issue tracking of the repository will be used. 

It's also a good opportunity to set the purpose of this question. Do we need to know how the project will track issues or do we need to study the existing tickets and technical debt as part of our evaluation process?

Reply:
Thanks for the comment, @skounis. I have attempted to add more context. Let me know if this addresses your points or you would like to see additional changes/updates. Also feel free to use the "suggestion" option in the comment to provide suggested language.